<launch>
	<arg name="doDebug" default="false" />
	<arg if="$(arg doDebug)" name="launchPrefix" value="xterm -e gdb -args" />
	<arg unless="$(arg doDebug)" name="launchPrefix" value="" />

	<node pkg="victoria_perception" name="kmeans_action" type="kmeans_action_node" respawn="true" output="screen" launch-prefix="$(arg launchPrefix)" />

</launch>